src/meta/src/stream/stream_manager.rs

```diff
@@ -745,6 +736,28 @@ where
         Ok(())
     }
 }
+
+fn vnode_mapping_to_actor_mapping(
```

Comment:
Let's put this function along with `actor_to_vnode` and `actor_mapping_from_bitmaps`?

Reply:
moved